README.md

```diff
@@ -387,95 +387,17 @@ dscl . -read /Users/<username> GeneratedUID
-1. Enter a new password, then enter the same password in the **Verify** field.
-1. Select **Set Password**.
-1. Select **Quit Firmware Utility** to close the Firmware Password Utility.
-1. Select Restart or Shutdown from the Apple menu in the top-left corner.
-
-The firmware password will activate at next boot. To validate the password, hold `Alt` during boot - you should be prompted to enter the password.
-
-The firmware password can also be managed with the `firmwarepasswd` utility while booted into the OS. For example, to prompt for the firmware password when attempting to boot from a different volume:
-
-```console
-sudo firmwarepasswd -setpasswd -setmode command
-```
-
-To verify the firmware password:
-
-```console
-sudo firmwarepasswd -verify
-```
-
-A firmware password may be bypassed by a determined attacker or Apple, with physical access to the computer.
-
-<img width="750" alt="Using a Dediprog SF600 to dump and flash a 2013 MacBook SPI Flash chip to remove a firmware password, sans Apple" src="https://cloud.githubusercontent.com/assets/12475110/17075918/0f851c0c-50e7-11e6-904d-0b56cf0080c1.png">
-
-*Using a [Dediprog SF600](https://www.dediprog.com/product/sf600) to dump and flash a 2013 MacBook SPI Flash chip to remove a firmware password, sans Apple*
+You should check that firmware security settings are set to [Full Security](https://support.apple.com/en-au/guide/mac-help/mchl768f7291/mac) to prevent tampering with your OS. This is the default setting.
 
-As of macOS 10.15, the `firmwarepasswd` program has a new option `-disable-reset-capability`. According to [Apple's new Platform Security page](https://support.apple.com/en-gb/guide/security/sec28382c9ca/web), this effectively prevents any firmware password resets, even by Apple themselves:
+## FileVault
 
-> For users who want no one but themselves to remove their Firmware Password by software means, the -disable-reset-capability option has been added to the firmwarepasswd command-line tool in macOS 10.15. Before setting this option, users must to acknowledge that if the password is forgotten and needs removal, the user must bear the cost of the motherboard replacement necessary to achieve this.
+All Mac models with Apple silicon are encrypted by default. Enabling [FileVault](https://support.apple.com/en-au/guide/mac-help/mh11785/mac) makes it so that you need to enter a password in order to access the data on your drive. Your FileVault password acts as a firmware password as well.
 
-Newer Mac models (Mac Pro, iMac Pro, Macbook with TouchBar) with [Apple T2](https://en.wikipedia.org/wiki/Apple-designed_processors#Apple_T2) chips, which provide a secure enclave for encrypted keys, lessen the risk of EFI firmware attacks. See [this blog post](https://michaellynn.github.io/2018/07/27/booting-secure/) for more information.
+FileVault protects data at rest and hardens against someone with physical access stealing data or tampering with your Mac.
```

Comment:
How does it do this? There are some prior resources about the specific threats and mechanisms (cold boot attacks, filevault analysis paper, etc) which could help demonstrate the point.

Reply:
These details should also be checked if they’re still relevant to Apple Silicion chips as many attacks are outdated/ Intel related

Reply:
I changed it to describe exactly what it does. I looked back at the things that are linked currently and they're covered now mostly by new hardware features. [This one](https://blog.frizk.net/2016/12/filevault-password-retrieval.html) uses DMA, but Apple silicon Macs protect memory with an [IOMMU](https://support.apple.com/guide/security/direct-memory-access-protections-seca4960c2b5/web) where every device is prevented from seeing memory for any other device. 

The FileVault analysis paper is about 12 years out of date so I feel like it's not a great resource to link to. If there's a more recent one I'd be ok with linking that, otherwise I'd like to just stick with official documentation.

Evicting the hardware keys isn't relevant since the Secure Enclave has its own encrypted memory that it uses for sensitive things like FileVault keys, so the raw key itself never leaves the [Secure Enclave](https://support.apple.com/guide/security/secure-enclave-sec59b0b31ff/web).

I'd probably rather cover this stuff in a Hardware section, but lemme know your thoughts.